.github/workflows/validate_samples.yml

```diff
@@ -0,0 +1,25 @@
+name: Validate-Samples
+
+on:
+  push:
+    paths:
+      - 'data/*/TIDES/*'
+      - 'spec/*'
+  pull_request:
+    paths:
+      - 'data/*/TIDES/*'
+      - 'spec/*'
```

Comment:
>Remove all files except validate_samples.yml

Not sure why? this is just tiering off of other files that are already there - shouldn't be adding anything else.

Reply:
This PR makes changes also proposed in #100, but should be independent or "stacked" on that PR, otherwise you'll be resolving merge conflicts when trying to merge this PR. Additionally, all files except `validate_samples.yml` are irrelevant to the purpose of this PR!